vectorstores/alloydb/vectorestores.go

```diff
@@ -157,15 +157,26 @@ func (vs *VectorStore) SimilaritySearch(ctx context.Context, query string, _ int
 	operator := vs.distanceStrategy.operator()
 	searchFunction := vs.distanceStrategy.similaritySearchFunction()
 
-	columns := append(vs.metadataColumns, vs.contentColumn)
+	columns := []string{}
+	columns = append(columns, vs.contentColumn)
 	if vs.metadataJsonColumn != "" {
 		columns = append(columns, vs.metadataJsonColumn)
 	}
 	columnNames := strings.Join(columns, `, `)
-	whereClause := ""
+	whereQuerys := make([]string, 0)
```

Comment:
@kurtisvg Do you think we should continue to just support a string for making more complex filters or move to a map filter where we can only support equals

Reply:
My 2c is stick to strings for now and we can always introduce a class that makes it easier to convert to a string later if needed.